docs/api/puppeteer.jshandle.getproperty_2.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: JSHandle.getProperty_2
+---
+
+# JSHandle.getProperty() method
```

Comment:
where is this coming from?

Reply:
is it back? :D

Reply:
Yes, the type test were failing without the it being there. I am trying to understand why it never showed up before as it existed prior to this PR.